packages/table/src/table2.tsx

```diff
@@ -62,7 +62,7 @@ import {
     resizeRowsByTallestCell,
 } from "./resizeRows";
 import { compareChildren, getHotkeysFromProps, isSelectionModeEnabled } from "./table2Utils";
-import { TableBody2 } from "./tableBody2";
+import { TableBody } from "./tableBody2";
```

Comment:
Should the filename also be renamed?
```suggestion
import { TableBody } from "./tableBody";
```